composer.json

```diff
@@ -3,8 +3,8 @@
     "description": "Shell completion for Symfony Console based scripts",
     "license": "MIT",
     "require": {
-        "symfony/console": "^2.5|3.*",
-        "symfony/process": "^2.3|3.*"
+        "symfony/console": "^2.5|^3.0|^4.0",
+        "symfony/process": "^2.3|^3.0|^4.0"
     },
     "require-dev": {
         "symfony/console": "2.8.*"
```

Comment:
I am wondering why it is fixed to 2.8.x in dev?

Reply:
IIRC tests would fail with a newer version as there were slight differences in the generated options. So updating this would require to adapt the tests.

Reply:
so does this mean the component will/may not work properly with `symfony/console != 2.8.*`?

Reply:
I will work, but there should be separate fixtures for different `symfony/console` versions or the test should exclude some of the built-in messages to be version independent.